### DISCUSSION TITLE
Updating records

### DISCUSSION THREAD

**Discussion by CNSam** [oddvalue/laravel-drafts#2] [Q&A] [answered]
Hi there,

So I have implemented the package into a simple 'pages' table.

I have gone ahead and created a page, i've then gone into an edit screen for the page and changed the page title...I then use the logic $page->create(['title' => 'new title']);

By my understanding, this should then create a new record in the database with the same UUID and mark that as is_current and is_published, it should also take care of unmarking the previous version as 'is_current' or am I misunderstanding the package here?

**Comment by oddvalue** [comment, marked as answer]
If you're trying to update a record then you should use the `update` method rather than `create` since `create` will create a new record.
